packaging/ios/build.sh

```diff
@@ -45,9 +45,9 @@ fi
 # Which versions are we building against?
 SQLITE_VERSION=3090200
 LIBXML2_VERSION=2.9.2
-LIBXSLT_VERSION=1.1.28
-LIBZIP_VERSION=0.11.2
-LIBGIT2_VERSION=0.26.0
+LIBXSLT_VERSION=1.1.32
+LIBZIP_VERSION=1.2.0
+LIBGIT2_VERSION=v0.26.0
```

Comment:
as we are updating things - did you check if 0.27 would work?
Otherwise we might at least look at 0.26.3 to get the bug fixes